airflow/providers/microsoft/azure/sensors/cosmos.py

```diff
@@ -31,12 +31,12 @@ class AzureCosmosDocumentSensor(BaseSensorOperator):
 
     .. code-block::
 
-    azure_cosmos_sensor = AzureCosmosDocumentSensor(
-        database_name="somedatabase_name",
-        collection_name="somecollection_name",
-        document_id="unique-doc-id",
-        azure_cosmos_conn_id="azure_cosmos_default",
-        task_id="azure_cosmos_sensor")
+        azure_cosmos_sensor = AzureCosmosDocumentSensor(
+            database_name="somedatabase_name",
+            collection_name="somecollection_name",
+            document_id="unique-doc-id",
+            azure_cosmos_conn_id="azure_cosmos_default",
+            task_id="azure_cosmos_sensor")
```

Comment:
```suggestion
            task_id="azure_cosmos_sensor",
        )
```
nit: this might look even better